client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -49,11 +56,20 @@ class _OnboardingPageState extends State<OnboardingPage> {
   }
 
   Future<void> _toNextPage() async {
-    await _pageController.nextPage(duration: _animationDuration, curve: _animationCurve);
+    await _pageController.nextPage(
+      duration: _animationDuration,
+      curve: _animationCurve,
+    );
   }
 
-  void _onDone() {
-    // Return true because onboarding is completed
-    Navigator.of(context).pop(true);
+  void _onDone() async {
```

Comment:
nitpick (code): `Future<void>` as method is async

Reply:
There's a linter rule for this that hasn't been enabled but I agree.

Reply:
There's on my normal work, so it's now also enabled in my eyes/brain-linter 😅